packages/react-avatar/src/components/Badge/Badge.types.tsx

```diff
@@ -21,6 +21,21 @@ export interface StatusProps extends ComponentProps, React.HTMLAttributes<HTMLEl
 
   /** The pre-defined state values which can be consumed directly. */
   state?: 'success' | 'info' | 'warning' | 'error' | 'unknown';
+
+  /** Style tokens */
+  tokens?: BadgeTokens;
```

Comment:
Question: I think I teetered between `ButtonTokens` and `ButtonTokenSet`, and landed on the latter because the base type is `TokenSet` I think. What's the right name here?

Reply:
I don't have a strong preference either way, but we definitely should be consistent between components. If button uses `ButtonTokenSet` then I'll rename these to match: `AvatarTokenSet` and `BadgeTokenSet`